### PR TITLE
Api v3 Routes

### PR DESCRIPTION
**Pull request by paglias**
cc @crookedneighbor

### CODE REVIEW

File: website/src/libs/api-v3/setupRoutes.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+fs
+  .readdirSync(CONTROLLERS_PATH)
+  .filter(fileName => fileName.match(/\.js$/))
+  .filter(fileName => fs.statSync(CONTROLLERS_PATH + fileName).isFile())
+  .forEach((fileName) => {
+    let controller = require(CONTROLLERS_PATH + fileName); // eslint-disable-line global-require
+
+    _.each(controller, (action) => {
+      let {method, url, middlewares, handler} = action;
```
Comment (crookedneighbor) [inline website/src/libs/api-v3/setupRoutes.js:16]: I love this syntax.

**Comment by crookedneighbor** [comment]
Are we going to implement API-doc?

**Comment by paglias** [comment]
Unless we have a better option i'd say yes, that's probably going to be my next pr

**Comment by crookedneighbor** [comment]
Awesome, can't wait

**Pull request merged**